config/mycnf/master_mariadb100.cnf

```diff
@@ -21,3 +24,21 @@ innodb_stats_persistent = 0
 # a master that becomes unresponsive.
 rpl_semi_sync_master_timeout = 1000000000000000000
 rpl_semi_sync_master_wait_no_slave = 1
+
+
+character_set_server = utf8
+collation_server = utf8_general_ci
```

Comment:
Does 10.0 not have utf8mb4?

Reply:
Actually for all the configs, should it use `utf8mb4`?

Reply:
It does, but it's very slow, and the upgrade path is not straight-forward. It's more clear-cut in 8.0 because `utf8mb4` is actually faster and mb3 is deprecated. References:

https://mysqlserverteam.com/sushi-beer-an-introduction-of-utf8-support-in-mysql-8-0/
https://mysqlserverteam.com/mysql-8-0-when-to-use-utf8mb3-over-utf8mb4/

Note the part about row-format=DYNAMIC: it is not true for all the versions we support, which would create an issue indexing varchar(255) columns.

As a general statement, this PR moves Vitess to "setting only the minimum MySQL configuration variables for correct operation, relying on the default provided by the MySQL flavor".